src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -679,7 +679,7 @@ public abstract class Avx2 : Avx
         public static Vector256<double> Permute4x64(Vector256<double> value, byte control) { throw null; }
         public static Vector256<int> PermuteVar8x32(Vector256<int> left, Vector256<int> mask) { throw null; }
         public static Vector256<uint> PermuteVar8x32(Vector256<uint> left, Vector256<uint> mask) { throw null; }
-        public static Vector256<float> PermuteVar8x32(Vector256<float> left, Vector256<float> mask) { throw null; }
+        public static Vector256<float> PermuteVar8x32(Vector256<float> left, Vector256<int> mask) { throw null; }
```

Comment:
Or, perhaps, `control`. This would be the same as a name of immediate parameters in `Permute4x64`. To unify naming we should check the names for all shuffles and other permutes as well.

Reply:
Agree.